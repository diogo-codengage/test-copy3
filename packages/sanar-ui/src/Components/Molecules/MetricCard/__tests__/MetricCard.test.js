import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESMetricCard from '../'

const mock = {
    img:
        'https://images.vexels.com/media/users/3/157872/isolated/preview/8d20eee691b54d23d865a69f08a40cd7---cone-terra-b--sica-by-vexels.png',
    status: 'danger',
    description: 'Horas estudadas na semana',
    badge: '25 de 50',
    doubt:
        'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
    title: 'ES METRIC CARD'
}

it('renders correctly', () => {
    const component = (
        <ESMetricCard
            img={mock.img}
            status={mock.status}
            description={mock.description}
            badge={mock.badge}
            style={{ width: '320px' }}
            doubt={mock.doubt}
            title={mock.title}
        />
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
