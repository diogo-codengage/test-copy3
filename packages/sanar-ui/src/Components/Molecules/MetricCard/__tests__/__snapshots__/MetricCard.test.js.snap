// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-card es-card es-metric-card ant-card-bordered ant-card-small"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        ES METRIC CARD
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div
      className="es-metric-card__content"
    >
      <img
        alt="Horas estudadas na semana"
        src="https://images.vexels.com/media/users/3/157872/isolated/preview/8d20eee691b54d23d865a69f08a40cd7---cone-terra-b--sica-by-vexels.png"
      />
      <div
        className="es-metric-card-badge es-metric-card-badge--danger"
      >
        <strong>
          25 de 50
        </strong>
      </div>
      <span
        aria-label={null}
        className="ant-typography"
        style={
          Object {
            "WebkitLineClamp": null,
          }
        }
      >
        <strong>
          Horas estudadas na semana
        </strong>
      </span>
      <i
        aria-label="icon: question-circle"
        className="anticon anticon-question-circle es-metric-card__doubt"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <svg
          aria-hidden="true"
          className=""
          data-icon="question-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
          />
          <path
            d="M623.6 316.7C593.6 290.4 554 276 512 276s-81.6 14.5-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56.1 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.1 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5.1-39.3-17.1-76-48.3-103.3zM472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0z"
          />
        </svg>
      </i>
    </div>
  </div>
</div>
`;
